editoast/src/main.rs

```diff
@@ -385,21 +386,22 @@ async fn import_rolling_stock(
 ) -> Result<(), Box<dyn Error + Send + Sync>> {
     for rolling_stock_path in args.rolling_stock_path {
         let rolling_stock_file = File::open(rolling_stock_path)?;
-        let mut rolling_stock: RollingStockModel =
+        let rolling_stock: RollingStockForm =
```

Comment:
```suggestion
        let rolling_stock: Changeset<RollingStockModel> =
```

The Forms are supposed to be the payloads of the endpoints, there's no need to tie them to cli commands.